BioArchLinux/orange/PKGBUILD

```diff
@@ -1,8 +1,7 @@
 # Maintainer: Kiri <kiri@vern.cc>
 
-pkgname='python-orange3'
+pkgname=orange
 _module=Orange3
-_pkgname=orange
 pkgver=3.35.0
 pkgrel=2
 pkgdesc="Orange, a component-based data mining framework."
```

Comment:
see archwiki, `Orange, ` should be removed

Reply:
https://wiki.archlinux.org/title/PKGBUILD#pkgdesc

Reply:
yep，i removed it